ui-spacetimechart/src/stories/options.stories.tsx

```diff
@@ -0,0 +1,197 @@
+import React, { type FC, useContext, useState } from 'react';
+
+import type { Meta } from '@storybook/react';
+import cx from 'classnames';
+import FileSaver from 'file-saver';
+
+import { MouseTracker } from './lib/components';
+import { OPERATIONAL_POINTS, PATHS } from './lib/paths';
+import {
+  MAX_X_ZOOM,
+  MAX_Y_ZOOM,
+  MIN_X_ZOOM,
+  MIN_Y_ZOOM,
+  X_ZOOM_LEVEL,
+  Y_ZOOM_LEVEL,
+} from './lib/utils';
+import { SpaceTimeChart, PathLayer } from '../';
+import { CanvasContext } from '../lib/context';
+import { type Point } from '../lib/types';
+import { getDiff } from '../utils/vectors';
+
+import './lib/tailwind-mockup.css';
+
+const ScreenshotButton: FC = () => {
+  const { captureCanvases } = useContext(CanvasContext);
+
+  return (
+    <div
+      style={{
+        position: 'absolute',
+        right: 10,
+        bottom: 10,
+      }}
+    >
+      <button
+        onClick={() =>
+          captureCanvases().then((blob) => FileSaver.saveAs(blob, 'space-time-chart.png'))
+        }
```

Comment:
is there a reason to use `FileSaver` instead of the canvas method `toDataURL()` ?

Reply:
Using `toDataURL` allows opening the image in a new tab, but it's not enough to download the image as a local file (which was what I was aiming at).

And historically, saving a local file from a browser is a bit tricky (using an hidden `a` tag in the DOM, and differently in some browsers). So, I used the most used JS lib to handle that, [file-saver](https://www.npmjs.com/package/file-saver).

Reply:
ok that was what I expected to see (add an hidden `a`)
thanks

Reply:
If you prefer that we cut the dependency, it's possible. But it's quite a light one, and it relieves us from maintaining some tricky (short, but still tricky) code that handles something `file-saver` does better.

Reply:
ok :)